tests/test_busybox.py

```diff
@@ -48,6 +48,42 @@ def test_busybox_image_size(
     )
 
 
+@pytest.mark.parametrize(
+    "container_per_test", [BUSYBOX_CONTAINER], indirect=True
+)
+def test_busybox_links(container_per_test):
+    # Ensure all commands in /bin are links to /usr/bin/busybox
+    container_per_test.connection.run_expect(
```

Comment:
This is not a mutating test, so no need to use the costly `_per_test` variant.

```suggestion
def test_busybox_links(auto_container):
    """Ensure all binaries in :file:`/bin` are links to :file:`/usr/bin/busybox`."""
    auto_container.connection.run_expect(
```

Reply:
Updated accordingly